.travis.yml

```diff
@@ -14,15 +14,6 @@ matrix:
   fast_finish: true
   include:
 
-    - os: linux
-      dist: xenial
```

Comment:
Would it be possible to isolate all Travis related changes in a single commit, so we can revert it in a isolated way if we had to go back ?

Reply:
See #3744